### ISSUE TITLE
Imported directives on enum values are not included in the output

### ISSUE THREAD

**Issue by trevorr** [ardatan/graphql-import#484]
Imported directives on enum values are not included in the output with v0.7.1 (and 1.0.0-beta.1).

Input files:

```graphql
# schema.graphql
# import * from 'directive.graphql'

schema {
  query: Query
}

type Query {
  color: ColorComponent @test
}

enum ColorComponent {
  RED @enumValue(string: "R")
  GREEN @enumValue(string: "G")
  BLUE @enumValue(string: "B")
}
```

```graphql
# directive.graphql
directive @enumValue(
  string: String
) on ENUM_VALUE

directive @test on FIELD_DEFINITION
```

Expected output:

```
type Query {
  color: ColorComponent @test
}

enum ColorComponent {
  RED @enumValue(string: "R")
  GREEN @enumValue(string: "G")
  BLUE @enumValue(string: "B")
}

directive @test on FIELD_DEFINITION

directive @enumValue(string: String) on ENUM_VALUE
```

Actual output:

```
type Query {
  color: ColorComponent @test
}

enum ColorComponent {
  RED @enumValue(string: "R")
  GREEN @enumValue(string: "G")
  BLUE @enumValue(string: "B")
}

directive @test on FIELD_DEFINITION
```

The `@enumValue` directive is not included in the output, which causes the schema to fail to build (e.g. with `buildASTSchema`). This appears to be caused by a straightforward omission in `collectNewTypeDefinitions`.

**Comment by trevorr** [comment]
I've created a [fix against 0.7.1](https://github.com/ardatan/graphql-import/compare/v0.7.1...trevorr:v0.7.1-enum-value-directive?expand=1). However, there doesn't appear to be a branch for the release version (master is past 1.0.0-beta.1). Would it be possible to create a 0.7.2 release with this small change? 1.0 seems massively reorganized.

**Comment by ardatan** [comment]
Available in 1.0.0!
